### ISSUE TITLE
[3.5.0-RC1] Extending form_submit.html5

### ISSUE THREAD

**Issue by ausi** [contao/core#7854]
In Contao 3.4 it was possible to extend the *form_submit* template, e.g.:

```html+php
<?php $this->extend('form_submit') ?>

<?php $this->block('field') ?>
	<button type="submit" ...
<?php $this->endblock() ?>
```

This does’t work anymore in 3.5.0-RC1 because of b3b993a0ef38f86cd30240647cade14a3b792d82.

We could fix this by wrapping the buttons with `<?php $this->block('field'); ?>`:

```html+php
<?php if ($this->tableless): ?>

  <div class="submit_container<?php if ($this->class) echo ' ' . $this->class; ?>">
    <?php $this->block('field'); ?>
      <?php if ($this->src): ?>
        <input type="image" src="<?= $this->src ?>" id="ctrl_<?= $this->id ?>" class="submit<?php if ($this->class) echo ' ' . $this->class; ?>" title="<?= specialchars($this->slabel) ?>" alt="<?= specialchars($this->slabel) ?>"<?= $this->getAttributes() ?>>
      <?php else: ?>
        <input type="submit" id="ctrl_<?= $this->id ?>" class="submit<?php if ($this->class) echo ' ' . $this->class; ?>" value="<?= specialchars($this->slabel) ?>"<?= $this->getAttributes() ?>>
      <?php endif; ?>
    <?php $this->endblock(); ?>
  </div>

<?php else: ?>

  <tr class="<?= $this->rowClass ?>">
    <td class="col_0 col_first">&nbsp;</td>
    <td class="col_1 col_last">
      <div class="submit_container">
        <?php $this->block('field'); ?>
          <?php if ($this->src): ?>
            <input type="image" src="<?= $this->src ?>" id="ctrl_<?= $this->id ?>" class="submit<?php if ($this->class) echo ' ' . $this->class; ?>" title="<?= specialchars($this->slabel) ?>" alt="<?= specialchars($this->slabel) ?>"<?= $this->getAttributes() ?>>
          <?php else: ?>
            <input type="submit" id="ctrl_<?= $this->id ?>" class="submit<?php if ($this->class) echo ' ' . $this->class; ?>" value="<?= specialchars($this->slabel) ?>"<?= $this->getAttributes() ?>>
          <?php endif; ?>
        <?php $this->endblock(); ?>
      </div>
    </td>
  </tr>

<?php endif; ?>
```

**Comment by Toflar** [comment]
:+1:

**Comment by leofeyer** [comment]
Fixed in 465cb72c01badbaf6e7273790ab68e25005c7e78.
